sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -572,14 +571,16 @@ public Mono<Response<ChatCompletions>> getChatCompletionsWithResponse(String dep
      * </p>
      * <!-- src_embed
      * com.azure.ai.openai.OpenAIAsyncClient.getChatCompletionsStream#String-ChatCompletionsOptionsMaxOverload -->
+     *
      * <pre>
-     * openAIAsyncClient.getChatCompletionsStreamWithResponse&#40;deploymentOrModelId, new ChatCompletionsOptions&#40;chatMessages&#41;,
-     *                 new RequestOptions&#40;&#41;.setHeader&#40;&quot;my-header&quot;, &quot;my-header-value&quot;&#41;&#41;
-     *         .subscribe&#40;
-     *                 response -&gt; System.out.print&#40;response.getValue&#40;&#41;.getId&#40;&#41;&#41;,
-     *                 error -&gt; System.err.println&#40;&quot;There was an error getting chat completions.&quot; + error&#41;,
-     *                 &#40;&#41; -&gt; System.out.println&#40;&quot;Completed called getChatCompletionsStreamWithResponse.&quot;&#41;&#41;;
+     * openAIAsyncClient
+     *     .getChatCompletionsStreamWithResponse&#40;deploymentOrModelId, new ChatCompletionsOptions&#40;chatMessages&#41;,
+     *         new RequestOptions&#40;&#41;.setHeader&#40;&quot;my-header&quot;, &quot;my-header-value&quot;&#41;&#41;
+     *     .subscribe&#40;response -&gt; System.out.print&#40;response.getValue&#40;&#41;.getId&#40;&#41;&#41;,
+     *         error -&gt; System.err.println&#40;&quot;There was an error getting chat completions.&quot; + error&#41;,
+     *         &#40;&#41; -&gt; System.out.println&#40;&quot;Completed called getChatCompletionsStreamWithResponse.&quot;&#41;&#41;;
      * </pre>
+     *
```

Comment:
The `getChatCompletionsStreamWithResponse` API is handwritten code via partial-update.

There is a diff of newline before/after `<pre>` tag.

Formatting in emitter would add the newline, and codesnippet would delete the new line.

So the result is that either codesnippet verify report error, or codegen verify report error.

Reply:
For generated classes, I wonder if we should surround the codesnippet tags with the formatter on and off comment

Reply:
Does eclipse format configure support such formatter on/off comment?

There could be 2 solutions I can think of
1. tweak the configure to avoid the newline
2. disable format on Javadoc, go back to rely on internal line-wrap logic (that would only format the Javadoc of generated class/method, but should leave the Javadoc on handwritten method untouched -- if java-parser does not touch them) @haolingdong-msft would java-parser modify Javadoc?

@alzimmermsft Does current eclipse format configure in SDK repo format Javadoc?

Reply:
The Eclipse format configuration in azure-sdk-for-java is slightly different than the one in Autorest.java where it has Javadoc, and all comment, formatting off in the assumption that hand-written comments are formatted as we want them. It's enabled in Autorest.java given that the code there is generated.

Eclipse has format [on](https://github.com/Azure/azure-sdk-for-java/blob/main/.vscode/eclipse-format-azure-sdk-for-java.xml#L29) and [off](https://github.com/Azure/azure-sdk-for-java/blob/main/.vscode/eclipse-format-azure-sdk-for-java.xml#L224) tags that could be used to wrap this custom methods to disable formatting.

I think there is also a third option available, which is wrapping any non-generated code in the on and off tags when running partial update.